Python/crossfire/tests/test_client.py

```diff
@@ -137,3 +153,80 @@ def test_client_load_states_raises_format_error(client_with_token):
         mock.return_value.json.return_value = {"data": []}
         with raises(UnknownFormatError):
             client_with_token.states(format="parquet")
+
+
+def test_client_load_states_as_dictionary(client_with_token):
+    with patch("crossfire.client.get") as mock:
+        mock.return_value.json.return_value = {"data": fake_cities_api_row()}
+        states = client_with_token.states(format="dict")
+        mock.assert_called_once_with(
+            "https://api-service.fogocruzado.org.br/api/v2/states",
+            headers={"Authorization": "Bearer 42"},
+        )
+        assert len(states) == 1
+        assert states[0]["name"] == "Rio de Janeiro"
+
+
+def test_client_load_cities(client_with_token):
+    with patch("crossfire.client.get") as mock:
+        mock.return_value.json.return_value = {"data": fake_cities_api_row()}
+        cities = client_with_token.cities()
+        mock.assert_called_once_with(
+            "https://api-service.fogocruzado.org.br/api/v2/cities?",
+            headers={"Authorization": "Bearer 42"},
+        )
+        assert cities[0].items() == fake_cities_api_row()[0].items()
+        assert cities[0]["name"] == "Rio de Janeiro"
+
+
+def test_client_load_cities_as_dictionary(client_with_token):
```

Comment:
A gente precisa desses dois testes? Eles me parecem testar a mesma coisa.

```suggestion
def test_client_load_cities(client_with_token):
```